packages/compass-e2e-tests/tests/logging.test.js

```diff
@@ -3,13 +3,19 @@ const { expect } = require('chai');
 const { beforeTests, afterTests } = require('../helpers/compass');
 
 function cleanLog(log) {
-  log = JSON.parse(
+  let cleanedLog = JSON.parse(
     JSON.stringify(log).replace(
       /(MongoDB( |\+|%20)Compass)(( |\+|%20)\w+)+/g,
       '$1'
     )
   );
-  for (const entry of log) {
+  cleanedLog = cleanedLog.filter(
+    (logEntry) =>
+      // Remove server heartbeat logs as they can happen a varying amount of
+      // times depending on how long tests are taking.
+      logEntry.id !== 1_001_000_022
+  );
```

Comment:
There’s already a check below – can you remove that, and expand this one to also cover `1_001_000_023` (heartbeat failure)?

Reply:
Since the heartbeat failure indicates a lack of connection or socket exception, wouldn't it make sense to fail the tests if we run into one? Since the machines for these tests, for the most case, are on the same machine running the tests I feel a heartbeat failure would probably not that likely?

Removed the check 👍 

https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-discovery-and-monitoring-monitoring.rst